### PR TITLE
KAADO-34: Create view card type page

### PR DESCRIPTION
**Pull request by nasudon**
- Create view card type page
- Add info and danger color to theme
- edit docker compose to run app outside docker

### CODE REVIEW

File: src/app/type/ListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use client';
```
Comment (tsoibet) [inline src/app/type/ListItem.tsx]: 'use client' may not be needed here.

File: src/app/type/page.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <div className="w-96 min-h-screen p-4 grid grid-rows-[auto_1fr] gap-4">
+            <TopNav />
+            <div className="text-center">
+                <div className="text-3xl font-bold m-4">Card Types</div>
+                <div className="py-12 w-52 mx-auto text-justify">
+                    {types && types.map((type) => <ListItem key={type._id} type={type} />)}
+                </div>
+                <Link href="/type/new">
+                    <div className="w-52 mt-4 mx-auto py-2 border rounded-lg text-lg">
+                        {' '}
```
Comment (tsoibet) [inline src/app/type/page.tsx]: Do we need this?

File: src/components/icons/TrashBinIcon.tsx
Comment (tsoibet) [inline src/components/icons/TrashBinIcon.tsx:1]: Trash icon is not used at all.

File: src/app/type/ListItem.tsx (hunk 1 above)
Comment (nasudon) [reply src/app/type/ListItem.tsx]: Updated: https://github.com/tsoibet/kaado/pull/7/commits/22eb888a2db1d2594e87eea92642811ca86efdae

File: src/app/type/page.tsx (hunk 2 above)
Comment (nasudon) [reply src/app/type/page.tsx]: Updated: https://github.com/tsoibet/kaado/pull/7/commits/22eb888a2db1d2594e87eea92642811ca86efdae

File: src/components/icons/TrashBinIcon.tsx
Comment (nasudon) [reply src/components/icons/TrashBinIcon.tsx:1]: Deleted: https://github.com/tsoibet/kaado/pull/7/commits/22eb888a2db1d2594e87eea92642811ca86efdae

**Review by tsoibet — APPROVED** [review]
LGTM

**Pull request merged**